lintlizard/__init__.py

```diff
@@ -37,17 +37,18 @@ def version_command(self) -> Command:
 
 
 TOOLS = [
+    # Python source formatters
+    CommandTool('mypy'),
     CommandTool('flake8', default_files=()),
     CommandTool(
         'isort', run_params=('-c',), fix_params=(), default_files=('.',)
     ),
-    CommandTool('mypy'),
     CommandTool(
-        'black',
-        run_params=('--check',),
-        fix_params=(),
-        default_files=('.',),
+        'black', run_params=('--check',), fix_params=(), default_files=('.',),
```

Comment:
This change is not compatible with black configuration used in LintLizard. You can try running LintLizard on itself - or even fix its own code.

Reply:
I haven't introduced any change here, the `run_params`, `fix_params` & `default_files` are the same as before no?

Reply:
aah nvm, silly me. got it 😄

Reply:
> You can try running LintLizard on itself - or even fix its own code.

thought it might be a good idea to introduce a CI step to put this to practice: https://github.com/closeio/lintlizard/pull/133/commits/d1f3e336b5b54b0353365a94768df803c8231cbb